TestsCalculator/SettingsPageTests.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using System.Collections.Generic;
+using System.Linq;
+using TestsCalculator.Pages;
+
+namespace TestsCalculator
+{
+    public class SettingsPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void SetUp()
+        {
+            driver = new ChromeDriver();
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            driver.Url = "http://127.0.0.1:8080/";
+
+            LoginPage loginPage = new LoginPage(driver);
+            loginPage.Login("test", "newyork1");
+
+            CalculatorPage calculatorPage = new CalculatorPage(driver);
+            calculatorPage.SettingsBtn.Click();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Close();
+        }
+
+        [Test]
+        public void CheckFieldNamesDisplayed()
+        {
+            //Arrange
+            SettingsPage settingsPage = new SettingsPage(driver);
+
+            //Assert
+            Assert.IsTrue(settingsPage.DateFormatText.Displayed);
+            Assert.IsTrue(settingsPage.NumberFormatText.Displayed);
+            Assert.IsTrue(settingsPage.CurrencyFormatText.Displayed);
```

Comment:
I think the better solution is to get elements via XPath and control name. Let's discuss it on a call.